app/controllers/hyrax/users_controller.rb

```diff
@@ -25,19 +25,33 @@ def show
     # @param query [String] the query string
     def search(query)
       clause = query.blank? ? nil : "%" + query.downcase + "%"
-      base = ::User.where(*base_query)
+      base = ::User
+      base = base.where(*Array.wrap(base_query)) if base_query.present?
```

Comment:
this feels a bit defensive. do we need both `Array.wrap` and the `#present?` check?

Reply:
The `Array.wrap` is because we need it to "splat" the return value (to keep the previous interface). The present is in a commit from months ago, I added it out of some necessity. Perhaps postgresql?

It definitely is a bit more defensive.

Reply:
I believe with the `*Array.wrap` I can remove the base_query.